### DISCUSSION TITLE
Do I need to redeploy altas when Windows 11 updates to 24h2 version in the future?

### DISCUSSION THREAD

**Discussion by haoboyuan1** [Atlas-OS/Atlas#1219] [Help & Support]
I am currently using the latest version of Windows 11 23h2 and the latest version of Altas.

**Comment by AudiEtoffe** [comment]
No one knows yet, but the authors have claimed that it is their goal to have .40 easily upgradable

**Comment by kennoxfr** [comment]
"Normaly" it's like that:
 - Anyone on v0.4 can upgrade on its release
    People who aren't on v0.4 will have to reinstall Windows to get v0.5

**Comment by Ast3risk-ops** [comment]
You'll find out when the time comes.
